integration/client/container_windows_test.go

```diff
@@ -0,0 +1,72 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package client
+
+import (
+	"golang.org/x/sys/windows"
+)
+
+// grantReadToEveryone adds a new entry in the ACL
+// of the file or folder, allowing "Everyone" to read
+// the contents. Inherit is set for all objects and
+// containers.
+func grantReadToEveryone(file string) error {
```

Comment:
I'm confused why you would need to do this manually. Does Docker do this and I didnt know for a passed in mount? This PR would imply that any folder share would not be read capable to ContainerUser by default right?